spec/test_app/db/schema.rb

```diff
@@ -45,6 +45,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.integer  "group_id"
+    t.string   "banner_url"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.